source/features/pinned-issues-update-time.tsx

```diff
@@ -43,12 +44,13 @@ async function init(): Promise<void | false> {
 	for (const pinnedIssue of pinnedIssues) {
 		const issueNumber = getPinnedIssueNumber(pinnedIssue);
 		const {updatedAt} = lastUpdated[api.escapeKey(issueNumber)];
-		select('.pinned-item-desc', pinnedIssue)!.append(
-			' • ',
-			<span className="color-fg-muted d-inline-block">
+		select('.pinned-item-desc', pinnedIssue)!.after(
+			<span className="text-small color-fg-muted">
 				updated <relative-time datetime={updatedAt}/>
 			</span>,
 		);
+
+		select('.pinned-item-desc', pinnedIssue)!.hidden = true;
```

Comment:
This changes the behavior of the feature. There's too much content up there and it causes the issue list to jump down in some cases.

I initially thought about dropping the feature, but I think it's useful more so than the native "user X opened on DATE" information.

The element is hidden, not removed, so it can be restored via CSS by the user.